frappe/core/doctype/user/user.py

```diff
@@ -121,16 +121,15 @@ def check_enable_disable(self):
 		if not cint(self.enabled):
 			self.a_system_manager_should_exist()
 			# disable notifications if the user has been disabled
-			enable_disable_notifications(self.name, enabled=False)
+			toggle_notifications(self.name, enable=False)
+		else:
+			# enable notifications if the user has been enabled
+			toggle_notifications(self.name, enable=True)
```

Comment:
You could make it even more compact

```suggestion
		if not cint(self.enabled):
			self.a_system_manager_should_exist()

		# toggle notifications based on the user's status
		toggle_notifications(self.name, enable=cint(self.enabled))
```

Reply:
Think its really worth the change?

Reply:
I'll leave it up to you, which is why I left it only as a comment lol

Reply:
From a clarity standpoint, it does separate the notifications into its own segment of code. Will make this change